projects.yml

```diff
@@ -5448,3 +5466,333 @@ rewrite_clj:
   url: https://github.com/clj-commons/rewrite-clj
   categories: [Data Transformation, Code Analysis]
   platforms: [clj, cljs]
+
+ambly:
+  name: Ambly
+  url: https://github.com/mfikes/ambly
+  categories: [REPLs]
+  platforms: [cljs]
+
+pyramid:
+  name: Pyramid
+  url: https://github.com/lilactown/pyramid
+  categories: [Graphs, EQL]
+  platforms: [clj, cljs]
+
+asami:
+  name: Asami
+  url: https://github.com/quoll/asami
+  categories: [Graphs]
+  platforms: [clj, cljs]
+
+clj_docker_client:
+  name: clj-docker-client
+  url: https://github.com/into-docker/clj-docker-client
+  categories: [Docker Integration]
+  platforms: [clj]
+
+contajners:
+  name: contajners
+  url: https://github.com/lispyclouds/contajners
+  categories: [Docker Integration]
+  platforms: [clj]
+
+clj_statecharts:
+  name: clj-statecharts
+  url: https://github.com/lucywang000/clj-statecharts
+  categories: [State Machines]
+  platforms: [clj, cljs]
+
+statecharts:
+  name: statecharts
+  url: https://github.com/fulcrologic/statecharts
+  categories: [State Machines]
+  platforms: [clj, cljs]
+
+obb:
+  name: obb
+  url: https://github.com/babashka/obb
+  categories: [Scripting]
+  platforms: [cljs]
+
+scittle:
+  name: scittle
+  url: https://github.com/babashka/scittle
+  categories: [ClojureScript Development, Web Frameworks]
+  platforms: [cljs]
+
+uri:
+  name: uri
+  url: https://github.com/lambdaisland/uri
+  categories: [Utilities]
```

Comment:
Change categories to: `[URI/URL]`. I don't know why there's a `Utilities` category -  think it was added by mistake.

Reply:
Thanks, I probably just used autocomplete and thought there was a "Utilities" instead of "Terminal/Filesystem/Namespace Utilities".